editor/js/compounds.js

```diff
@@ -306,7 +403,13 @@ function addNameIdentifier(node) {
   xhr.send(name);
 }
 
-// Shortcut to add an identifier by drawing.
+/**
+ * Displays the Ketcher drawing tool for defining compound identifiers. A
+ * callback is defined to create SMILES and MOLBLOCK-type identifiers; this
+ * callback is triggered upon submission from the Ketcher window.
```

Comment:
"submission of a molecular drawing" perhaps? Was a little confusing to me at first glance...

Reply:
Done.